src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/Quotable2.java

```diff
@@ -0,0 +1,8 @@
+package jdk.incubator.code.internal;
+
+import jdk.incubator.code.Quotable;
+import jdk.incubator.code.Quoted;
+
+public interface Quotable2 extends Quotable {
```

Comment:
A better name would be preferrable. Maybe `AccessibleQuotable` ?

Reply:
Yes, something more descriptive.